packages/apollo/src/commands/service/__tests__/check.test.ts

```diff
@@ -369,6 +372,11 @@ describe("service:check", () => {
 
     nock.disableNetConnect();
 
+    delete process.env.CI;
```

Comment:
re these lines, I think this is a fine solution. Not sure there's a more effective way to simulate this environment